application/templates/finished.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base/lr_base_layout.html" %}
+
+{% block content %}
+
+<div class="main" id="content">
+  {{ super() }}
+  <div class="inner-block">
+  <div class="deed-successful">
+  <h3 class="heading-large">Thank you for viewing your mortgage deed</h3>
+  </div>
+
+  <h2 class="heading-medium">What happens next?</h2>
+  <p>The online deed you've viewed is not the legal deed.</p>
+  <ul>
+  <li>
+  You should sign and return your paper mortgage deed following the instructions you have received from your conveyancer
+  </li>
+  <li>
+  Land Registry will then update your title register to include etc.
+  </li>
+  </ul>
+  <p>
+  The time this takes will vary depending on the complexity of the
+  transaction.
+  </p>
+  <div class="division-standard-top">
+  <ul>
+  <li><a href="#">What did you think of this service?</a> (Takes 30 seconds)</li>
+  <li><a href="#">More about the Land Registry</a></li>
```

Comment:
That's from the prototype, the links don't go anywhere it's more for the look and feel I suppose which I reckon is okay